testing/bdd/test_required_metadata.py

```diff
@@ -0,0 +1,12 @@
+import xml.etree.ElementTree as ET
```

Comment:
please could you rename this file `test_ebuttd_required_metadata.py` to distinguish it from the Part 3 case, and possibly in the future a Part 1 case?